enterprise/backend/src/metabase_enterprise/advanced_permissions/models/permissions/group_manager.clj

```diff
@@ -0,0 +1,56 @@
+(ns metabase-enterprise.advanced-permissions.models.permissions.group-manager
+  (:require [clojure.data :as data]
+            [clojure.set :refer [subset?]]
```

Comment:
could you just alias `clojure.set` to `set` rather than referring `subset?`? this is what's done in other usages of `clojure.set` on the backend (and we generally avoid `:refer` except for specific cases)